### PR TITLE
'Bored' additions to the adventure

### PR DESCRIPTION
**Pull request by versatgrant**
Please review this, @casey-collab

### CODE REVIEW

File: english/bored/bored.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"I'm bored."
```
Comment (casey-collab) [inline english/bored/bored.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/bored/bored.md (hunk 1 above)
Comment (versatgrant) [inline english/bored/bored.md]: Made it reachable, @casey-collab

**Comment by shersom** [comment]
Seems like this should have worked, @casey-collab.

**Comment by casey-collab** [comment]
Looks great! :ferris_wheel:

**Pull request merged**